src/test/ui/rfc1445/issue-61118-match-slice-forbidden-without-eq.rs

```diff
@@ -0,0 +1,19 @@
+// Issue 61118 pointed out a case where we hit an ICE during code gen:
+// the compiler assumed that PartialEq was always implemented on any
+// const used in a pattern context, but the code responsible for
```

Comment:
```suggestion
// `const` used in a pattern context, but the code responsible for
```

Reply:
mmm, I usually don't put something in `code format` when I'm not using the syntactic form and am instead using the word to describe a mechanism or broader pattern. Sort of like how one would not put "trait" in `code format` unless one were talking about a bit of code that actually used the `trait` keyword.

Reply:
Fair :)

Reply:
Would you prefer I use another word here? I did consider saying "constant" but then I worried that would then stray into the territory of describing a syntactic literal, and I need the reader to keep in mind that we are talking about uses of `const` items.

Maybe I should say it that way, "... that `PartialEq` was always implemented on any use of a `const` item in a pattern context, but ..." ?

Reply:
I wouldn't overthink it -- it was mostly a nit on my part. ;)
That said, your last suggestion makes sense.